### ISSUE TITLE
Disabling /demo due to XSS on 1.11.0

### ISSUE THREAD

**Issue by kyroskoh** [mapproxy/mapproxy#414]
I still receive XSS for 1.11.0 for /demo.

removed 

> demo:

 in line 2
```
services:
  wmts:
    restful: true
    kvp: true
  tms:
    use_grid_names: true
    origin: 'sw'
  wms:
    srs: ['EPSG:4326', 'EPSG:3414']
    image_formats: ['image/png']
```

I would like to disable "Welcome to MapProxy" page. How can I able to block the request?

I tried to use nginx to add in deny all; for location / but it doesn't work.

**Comment by Niarolf** [comment]
Hi,
At the top of the yaml you can remove the line "demo".

**Comment by olt** [comment]
Can you please be so kind and give us a few more details about the XSS, so that we can fix this for everyone?

**Comment by olt** [comment]
I got details about the XSS issue (incomplete fix for #322).
There is an updated fix with 420412aa in master and 436c8f4897 in 1.11.x branch.  MapProxy 1.11.1 release is out with a fix.

**Comment by kyroskoh** [comment]
Thanks for the fix.

Sorry for my late reply for assistance as I have been busy.

Yes, those are the escaping characters: 
&
<
>
"
'

In my XSS report via my vendor, they stated that it can be parsed via / too but I am not sure how u guys able to handle it.
